pkg/KubeArmorOperator/internal/controller/resources.go

```diff
@@ -234,6 +230,8 @@ func deploySnitch(nodename string, runtime string) *batchv1.Job {
 							},
 						},
 						SecurityContext: &corev1.SecurityContext{
+							RunAsUser:  &rootUser,
+							RunAsGroup: &rootUser,
```

Comment:
Do we need to explicitly specify it?

Reply:
yes we need to specify it explicitly. on redhat if it's not specified explicitly it will be running as a non-root user it might be true for some other platform too.